build/grunt.js

```diff
@@ -190,6 +190,7 @@ module.exports = function(grunt) {
       swf:   { cwd: 'node_modules/videojs-swf/dist/', src: 'video-js.swf', dest: 'build/temp/', expand: true, filter: 'isFile' },
       ie8:   { cwd: 'node_modules/videojs-ie8/dist/', src: ['**/**'], dest: 'build/temp/ie8/', expand: true, filter: 'isFile' },
       dist:  { cwd: 'build/temp/', src: ['**/**', '!test*'], dest: 'dist/', expand: true, filter: 'isFile' },
+      a11y:  { src: 'sandbox/descriptions.html.example', dest: 'sandbox/descriptions.test-a11y.html' }, // Can only test a file with a .html or .htm extension
```

Comment:
Then why ignore `html` files in the sandbox? Why not let people try them out as they are, and edit them or add others, but not commit them? Or otherwise move the existing examples into  an example directory, with the option to copy them into the sandbox where they'll be ignored. I'm missing something about the intention of the sandbox and the examples, I guess!